xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp

```diff
@@ -54,7 +54,7 @@
   ""
 
 #define RUNSCRIPT_POSTSCRIPT \
-        MODULE ".log('-->HTTP Python WSGI Interpreter Initialized<--', " MODULE ".LOGNOTICE)\n" \
+        MODULE ".log('-->HTTP Python WSGI Interpreter Initialized<--', " MODULE ".LOGINFO)\n" \
```

Comment:
Is this correct or should it be LOGWARNING?

Reply:
Correct